tl-tl/exchange.md

```diff
@@ -86,114 +88,116 @@ To create a wallet, do the following：
 | P2P                | 10333   | 20333   |
 | websocket          | 10334   | 20334   |
 
-### Generating Deposit Addresses
+### Pagbubuo ng mga Address sa Deposito
 
-A wallet can store multiple addresses. The exchange needs to generate a deposit address for each user. 
+Ang isang wallet ay maaaring mag-imbak ng maramihang mga address. Ang palitan ay kailangang bumuo ng isang addres sa deposito para sa bawat gumagamit.
+Mayroong dalawang mga paraan upang makabuo ng mga address ng deposito:
 
-There are two methods to generate deposit addresses: 
+- Kapag ang gumagamit sa deposito (NEO / NEO GAS) sa unang pagkakataon, ang programa ay dynamic na bumubuo ng isang NEO address. Ang kalamangan ay hindi na kailangan upang makabuo ng mga address sa mga nakapirming mga agwat ng oras, habang ang kawalan ay hindi mo maaaring i-backup ang wallet.
 
-- When the user deposit (NEO/NEO GAS) for the first time, the program dynamically generates a NEO address. The advantage is that there is no need to generate addresses at fixed time intervals, while the disadvantage is you can not backup the wallet.
 
-  To develop the program to dynamically generate addresses, use the NEO-CLI API  [getnewaddress Method](node/api/getnewaddress.html). The created address is returned.
+-Upang bumuo ng programa patungong dynamically binuong mga address,, gamitin ang NEO-CLI API  [getnewaddress Method](node/api/getnewaddress.html). Ang nagawa na address ay ibinalik.
+  
 
-- The exchange creates a batch of NEO addresses in advance. When the user charges (NEO/NEO GAS) for the first time, the exchange assigns a NEO address to him or her. The advantage is the convenience to backup the wallet, while the disadvantage is the need to generate NEO addresses manually.
-  To generate addresses in batch, run the NEO- CLI command `create address [n]`. The  addresses are exported automatically to the address.txt file.
-  [n] is optional. Its default value is 1. For example, to generate 100 addresses at a time, enter `create address 100`.
+- Ang palitan ay lumilikha ng batch ng NEO address nang maaga. Kapag ang mga singil ng gumagamit (NEO / NEO GAS) sa unang pagkakataon, ang palitan ay nagtatalaga ng isang NEO address sa kanya. Ang kalamangan ay ang kaginhawaan na i-backup ang wallet, habang ang kawalan ay ang pangangailangan upang bumuo ng mga address NEO nang manu-mano.
```

Comment:
Please improve this phrase `Kapag ang mga singil ng gumagamit (NEO / NEO GAS) sa unang pagkakataon`